build_library/build_image_util.sh

```diff
@@ -658,6 +658,7 @@ finish_image() {
       ${tmp_ignore} "${root_fs_dir}/var"
   sudo "${BUILD_LIBRARY_DIR}/gen_tmpfiles.py" --root="${root_fs_dir}" \
       --output="${root_fs_dir}/usr/lib/tmpfiles.d/base_image_etc.conf" \
+      --files="/usr/flatcar-etc" \
```

Comment:
How about just '/usr/etc'?

Reply:
Also possible :) I wanted to somehow imply that this is a copy of "upstream", maybe there are other words to use like `etc-reference`, how about that?

Reply:
Put it under `/usr/share/flatcar/etc` now